__tests__/components/lists/SessionsListVertical.test.tsx

```diff
@@ -1,27 +1,51 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { render, screen } from '@testing-library/react-native';
 import SessionsListVertical from '../../../components/lists/SessionsListVertical';
 import { Schedule } from '../../../mock/schedule';
 
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+      staleTime: Infinity,
+    },
+  },
+});
+
 jest.mock('expo-router');
 
 describe('SessionsListVertical', () => {
   it('renders SessionsListVertical component', () => {
-    render(<SessionsListVertical sessions={Schedule.data['2022-11-16']} bookmarked={false} />);
+    render(
+      <QueryClientProvider client={queryClient}>
```

Comment:
Can we make a custom render method that has this provider already wrapped around it so that we can avoid doing this every time we render the components in the tests? https://testing-library.com/docs/react-testing-library/setup#custom-render